config/model.js

```diff
@@ -0,0 +1,28 @@
+const db = require("../database/dbConfig");
+
+module.exports = {
+  add,
+  find,
+  findBy,
+  findById
+};
+
+function find() {
+  return db("users").select("id", "username");
+}
+
+function findBy(filter) {
+  return db("users").where(filter);
+}
+
+async function add(user) {
+  const [id] = await db("users").insert(user);
```

Comment:
I notice you are mixing promises and async in the helper methods here. I would pick one way and you it throughout to keep consistency